controllers/providers/assistant/gslb.go

```diff
@@ -189,32 +189,24 @@ func (r *GslbLoggerAssistant) InspectTXTThreshold(fqdn string, fakeDNSEnabled bo
 		log.Info().Msgf("Error contacting EdgeDNS server (%s) for TXT split brain record: (%s)", ns, err)
 		return err
 	}
-	var timestamp string
 	if len(txt.Answer) > 0 {
 		if t, ok := txt.Answer[0].(*dns.TXT); ok {
-			log.Info().Msgf("Split brain TXT raw record: %s", t.String())
-			timestamp = strings.Split(t.String(), "\t")[4]
+			timestamp := strings.Split(t.String(), "\t")[4]
 			timestamp = strings.Trim(timestamp, "\"") // Otherwise time.Parse() will miserably fail
-		}
-	}
-
-	if len(timestamp) > 0 {
-		log.Info().Msgf("Split brain TXT raw time stamp: %s", timestamp)
-		timeFromTXT, err := time.Parse("2006-01-02T15:04:05", timestamp)
-		if err != nil {
-			return err
-		}
-
-		log.Info().Msgf("Split brain TXT parsed time stamp: %s", timeFromTXT)
-		now := time.Now().UTC()
-
-		diff := now.Sub(timeFromTXT)
-		log.Info().Msgf("Split brain TXT time diff: %s", diff)
+			timeFromTXT, err := time.Parse("2006-01-02T15:04:05", timestamp)
+			if err != nil {
+				log.Info().Msgf("Split brain TXT [raw record: %s, raw timestamp: %s]", t.String(), timestamp)
+				return err
+			}
+			now := time.Now().UTC()
+			diff := now.Sub(timeFromTXT)
+			log.Info().Msgf("Split brain TXT [raw record: %s, raw timestamp: %s, parsed: %s, diff: %s]", t.String(), timestamp, timeFromTXT, diff)
```

Comment:
We can move this one to Debug i think. No need to log it on every heartbeat on Info level

Reply:
Are we going to utilize zerolog structure logging capabilities and bring more context into logging events?
In this case the code would look like:
```suggestion
			log.Info().
				Str("raw record", t.String()).
				Str("raw timestamp", timestamp).
				Str("parsed", timeFromTXT.String()).
				Str("diff", diff.String()).
				Msg("Split brain TXT")
```
or
```suggestion
			log.Info().Fields(map[string]interface{}{
				"raw record":    t.String(),
				"raw timestamp": timestamp,
				"parsed":        timeFromTXT.String(),
				"diff":          diff.String()}).
				Msg("Split brain TXT")
```